### PR TITLE
allow specification of UID and GID of the plex user

### PR DESCRIPTION
**Pull request by kalbasit**
plus a few fixes that I'll highlight as comments on the PR.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -38,12 +37,13 @@ if [ "${CHANGE_DIR_RIGHTS}" = true ]; then
   chmod -R g+rX /data
 fi
 
-#remove previous pid if it exists
-rm ~/Library/Application\ Support/Plex\ Media\ Server/plexmediaserver.pid
```
Comment (kalbasit) [inline start.sh]: This was moved down to the `if .. fi` block because it fails to execute if the Library is mounted over NFS.

File: start.sh [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+  usermod -G "`stat -c "%g" /data`" plex
 fi
 
-usermod -a -G ${GROUP} plex
+## initialize supervisor
+
+mkdir -p /config/logs/supervisor
+touch /supervisord.log
+touch /supervisord.pid
+chown plex: /supervisord.log /supervisord.pid
```
Comment (kalbasit) [inline start.sh]: This was moved under the UID/GID workflow to guarantee the correct permissions of the file.

File: start.sh [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+
+## set the correct permissions on /config unless SKIP_CHOWN_CONFIG was set.
 
 if [[ -z "${SKIP_CHOWN_CONFIG}" ]]; then
   CHANGE_CONFIG_DIR_OWNERSHIP=false
```
Comment (kalbasit) [inline start.sh]: @timhaak I did not want to break backward compatibility but why do we have both `CHANGE_CONFIG_DIR_OWNERSHIP` and `SKIP_CHOWN_CONFIG`? Is it okay with you to deprecate SKIP_CHOWN_CONFIG and echo a message to notify future removal?

**Comment by kalbasit** [comment]
@timhaak have you had a chance to look at this?

**Comment by timhaak** [comment]
Hi 

Sorry today is actually the first day in a while I've had time.

I think at the time the original pull request had one env variable and there was another. So to make it that I didn't break someone's build I included both.

Though agree we should simplify and make it one. 

I vote this one CHANGE_CONFIG_DIR_OWNERSHIP

?

**Comment by kalbasit** [comment]
@timhaak agree. I'll file a separate PR. Also do you want me to file the same PRs for docker-plex or you'll just cherry-pick yourself?

**Pull request merged**